src/app/pages/swap/components/swap-footer.tsx

```diff
@@ -0,0 +1,19 @@
+import { useFormikContext } from 'formik';
+
+import { InfoCardFooter } from '@app/components/info-card/info-card';
+import { PreviewButton } from '@app/components/preview-button';
+
+export function SwapFooter() {
```

Comment:
Think we could name better, more about what the content of this component is, rather than where it happens to be placed